src/main/java/pvs/app/entity/GitlabCommit.java

```diff
@@ -0,0 +1,68 @@
+package pvs.app.entity;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.Date;
+import java.util.Objects;
+
+@Entity
+@Data
```

Comment:
Using @Data for JPA entities is not recommended. It can cause severe performance and memory consumption issues.

```suggestion
@Getter
@Setter
@ToString
@RequiredArgsConstructor
```